src/global.css

```
@@ -6,12 +6,19 @@
   box-sizing: border-box;
 }
 
+:root {
+  --color-primary: #91c788;
+  --color-secondary: #ff8473;
+  --color-background-primary: white;
```

```suggestion
  --color-background-primary: #F4F4F4;
```
Almost white ;) Otherwise, looks good to me! 💯